src/Header.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const StyledNav = styled.nav`
```

Comment:
You could simply call this `Nav` imo. The `Styled` prefix just adds clutter from my perspective